### ISSUE TITLE
Training my model won't give me a good dialog

### ISSUE THREAD

**Issue by Frackup** [RasaHQ/rasa_nlu#881]
**Rasa NLU version** : 0.11.3

**Used backend / pipeline** : spacy_sklearn

**Operating system** : Ubuntu 16.04

**Issue**: Dialog training does not seem to work properly.

Hi all,

First of all, I try to implement a chatbot using Rasa with the french version provided by Spacy, so I will just start by showing here my config file and check with you if it's correctly set up.

**Content of configuration file** :
```
{
  "pipeline": [
    "nlp_spacy",
    "tokenizer_spacy",
    "intent_entity_featurizer_regex",
    "intent_featurizer_spacy",
    "intent_classifier_sklearn",
    "ner_crf",
    "ner_synonyms"
  ],
  "path" : "./models/nlu",
  "data" : "./data/train_file.json",
  "language" : "fr",
  "response_logs": "logs",
  "log_level": "INFO",
  "spacy_model_name": null
}

```

I completed my domain file with all my intents, entities, slots, actions and templates. Don't think it's interesting to show it, I guess I have correctly created and filled it.

Then I come to the main point, my story.md file. I created one, based on the given examples (eg : restaurant bot) and I put some different dialog within it (I show just a part of it):

```
## Dialogue_bonjour
* bonjour
  - utter_bonjour

## Dialogue_virement_initial
* bonjour
  - utter_bonjour
* realiserVirement
  - utter_demanderMontant
* realiserVirement{"montant": "500"}
  - slot{"montant": "500"}
  - utter_merki
  - utter_demanderCompteADebiter
* realiserVirement{"compteADebiter": "le compte de mon fils"}
  - slot{"compteADebiter": "le compte de mon fils"}
  - utter_merki
  - utter_demanderCompteACrediter
* realiserVirement{"compteACrediter": "le compte perso"}
  - slot{"compteACrediter": "le compte perso"}
  - utter_merki
  - utter_demanderConfirmationVirement
  - utter_confirmerOperationVirement
  - utter_autreAction
* aurevoir
  - utter_aurevoir
```

Then I train my model, and I have some strange behavior.
First if I ask to do a transfer (realiserVirement), it seems to work and the bot ask me for my transfer amount, once done it goes and asks me for the from where the transfer is done, but then, when I give the account (it's a text slot so I can give whatever I want, no verification), it's not responding anymore.

I then told me, maybe I uncorrectly created my story file, so I decide to go and try interactive learning, but each time I check the story I create during this exercise, it's exactly the same type of story than those I created myself.

And last point, if I don't put the bonjour and utter_bonjour in a single story, the bot is not likely to tell bonjour at the first launch when I tell him bonjour, why that?

If you need further files / details, don't hesitate, thank you.

**Comment by Frackup** [comment]
I just saw that in the restaurant bot, you have a story file containing 27K lines of code. Is this what's awaited to set up a chatbot? Is there a good tool to ease this creation? It this the reason my dialog is not working properly? (I have only 85 lines).

**Comment by wrathagom** [comment]
@Frackup probably best to re-create this question in the Rasa Core repo. Did you see the online learning tutorial?

**Comment by Frackup** [comment]
Hi,

Of course you're right it's definitely a core issue (and not nlu), I will recreate.

Yes I followed the tutorial 😞, but as said, all my inputs seem to be like the ones shown in the tutorial.

But I keep on looking for a solution and find that maybe when we come up with some intents and entities (a more complex chatbot than a moodbot), I have then to look deeper in the files like "restaurant_policy" and "bot.py" to adapt them to my case, do you think I am right?

Thanks.
Regards,
________________________________
De : Caleb M. Keller <notifications@github.com>
Envoyé : mercredi 7 mars 2018 15:07
À : RasaHQ/rasa_nlu
Cc : Alexandre; Mention
Objet : Re: [RasaHQ/rasa_nlu] Training my model won't give me a good dialog (#881)


Closed #881<https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FRasaHQ%2Frasa_nlu%2Fissues%2F881&data=02%7C01%7C%7Cd76447f63a114dbc733808d58434b79b%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636560284304490044&sdata=dUR6bxbOB8xlHmbyL3UC9vC0MxswdeZeilL%2Ft8uE4P0%3D&reserved=0>.
